assets/src/edit-story/components/fontPicker/test/fontPicker.js

```diff
@@ -97,12 +97,13 @@ describe('Font Picker', () => {
 
     // Roboto option should be visible and have a selected checkmark
     const selectedRobotoOption = getByRole('option', {
-      // The "accessible name" is derived by concatenating the accessible names
-      // of all children,  which in this case is an SVG with aria-label="Selected"
-      // and a plain text node with the font name. Thus this works!
-      name: 'Selected Roboto',
+      selected: true,
     });
     expect(selectedRobotoOption).toBeInTheDocument();
+    expect(selectedRobotoOption.lastChild).toHaveAttribute(
+      'font-family',
+      "'Roboto::MENU'"
+    );
```

Comment:
Does the node have this actual attribute? That's not intended to be valid long-term. We really should cleanup these errant attributes, that we're spraying all over the DOM nodes – by using S-C 5.1's [transient props](https://styled-components.com/docs/api#transient-props) or [`shouldForwardProp`](https://styled-components.com/docs/api#shouldforwardprop).

I think the test here should validate the actual style object via `.toHaveStyle`, which I believe we have as a validator.

Reply:
yup, does have that attribute, but happy to check the style instead. just trying to validate that this is using roboto for the integrity of the test without using the `name` selector.

I can most likely revert this test back to what it originally was if I alter the `ScrollList` in a way that it doesn't need to render the list labels (`recommended` & `recently used`) inside the list item